src/main/app/form/Form.test.ts

```diff
@@ -130,8 +130,8 @@ describe('Form', () => {
     expect(form.getParsedBody(body)).toStrictEqual({
       field: 'YES',
       dateField: {
-        day: '1',
-        month: '1',
+        day: '01',
```

Comment:
Still don't think this is right but I might not understand what the form is producing

Reply:
👍 I've refactored this so that the `fromApiFormat` will drop any padded zeros in 559b32b.